lib/eventsource-polyfill.js

```diff
@@ -1,5 +1,6 @@
 var EventSource = require('./eventsource')
 if (typeof window === 'object') {
   window.EventSource = window.EventSource || EventSource
+} else if (typeof global === 'object') {
+  global.EventSource = global.EventSource || EventSource
```

Comment:
True, I guess you have to cover for people doing `<script src='eventsource-polyfill.js'>` 😒